Sources/Receipt/ReceiptFeeTableViewCell.swift

```diff
@@ -28,6 +28,13 @@ final class ReceiptFeeTableViewCell: UITableViewCell {
         super.init(coder: aDecoder)
     }
 
+    func configure(configuration: (title: String, value: String)) {
+        textLabel?.text = configuration.title
+        textLabel?.accessibilityIdentifier = configuration.title
+        detailTextLabel?.text = configuration.value
+        detailTextLabel?.accessibilityIdentifier = configuration.value
```

Comment:
let's change it to "(configuration.title)_value" instead of configuration.value

Reply:
Thanks. This format looks good instead of the value which will be different based on the mock data.

Reply:
fixed